styles.css

```diff
@@ -0,0 +1,76 @@
+
```

Comment:
Please try to make your design as close as possible to Figma's. You could give your elements a margin so that they have more spacing towards the edge of the screen. As you can see in the following image, your social icons are not showing properly. You can give them a bit of margin so they can align themselves in a more organized way 🤓 

Yours
![image](https://user-images.githubusercontent.com/15807343/159564899-790bf641-266c-465e-9b1a-6a3e6ff274ba.png)

Figma's 
![image](https://user-images.githubusercontent.com/77142475/159115841-4e120b15-7002-43aa-812f-5774ddcfa81d.png)